ExchangeSharp/API/Exchanges/_Base/ExchangeAPI.cs

```diff
@@ -443,12 +235,12 @@ public void Dispose()
             {
                 disposed = true;
                 OnDispose();
-                Cache.Dispose();
+                Cache?.Dispose();
```

Comment:
Cache should never be null. Do you have a case where this is null?

Reply:
This nullref'd on me. No idea why

Reply:
That is very odd. Here it the definition:

```csharp
protected MemoryCache Cache { get; } = new MemoryCache();
```

Reply:
Can it dispose twice?

Reply:
I suppose it's possible. The Dispose in MemoryCache is wrapped in a try/catch so *shouldn't* be a problem.